tox.ini

```diff
@@ -0,0 +1,16 @@
+[tox]
+envlist = py27,pep8
+skipsdist = True
+
+[testenv]
+deps = 
+    mock
+    nose
+    -r{toxinidir}/requirements.txt
```

Comment:
A space is not allowed in this case (I believe this is related to how tox parses deps, but I haven't actually looked into that much).